charts/install-secrets-manager/README.md

```diff
@@ -40,7 +40,7 @@ The following tables lists the configurable parameters of the install-secrets-ma
 | `image.repository`                        | The repository of the docker image for secret-manger. There is no official image so you must build your own.                                                                                                                                              | `nil`                                                         |
 | `image.version`                        | The docker image tag for the version of the container you would like kubernetes to pull down                                                                                                                                              | `nil`                                                         |
 | `vaultLocation`                        | URI path to your vault instance registry                                                                                                                                              | `nil`                                                         |
-| `vaultVersion`                        | vault key value version registry                                                                                                                                              | `kv2`                                                         |
+| `vaultVersion`                        | vault key value version registry                                                                                                                                              | `kv1`                                                         |
```

Comment:
If we decide to go through with changing the default, we'll need to update the `values.yaml` too.

Reply:
I always set kv1